.github/workflows/update-fpl-data.yml

```
@@ -0,0 +1,109 @@
+  update-fpl-data:
+    name: Update Fantasy Premier League Data
+    runs-on: ubuntu-latest
+    env:
+      RESOURCE_GROUP: powellrhys-sql-server-rg
+      SQL_SERVER: powellrhys-sqlserver
+      FIREWALL_RULE_NAME: update-data-github-action-${{ github.run_id }}
+
+    steps:
+      # Check out codebase
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)